src/styles/main.scss

```diff
@@ -22,57 +22,78 @@ body {
   user-select: none;
 
   &--2 {
+    animation: moving 0.3s forwards;
```

Comment:
You can move it directly to `field-cell` and overwrite `animation-duration` to 0.5s in `&--2048`